src/otx/v2/adapters/torch/lightning/__init__.py

```diff
@@ -2,3 +2,24 @@
 
 # Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
+
+AVAILABLE = True
+VERSION = None
+DEBUG = None
+
+try:
+    import pytorch_lightning as pl
+    VERSION = pl.__version__
+    import os
+
+    os.environ["FEATURE_FLAGS_OTX_VISUAL_PROMPTING_TASKS"] = "1"
```

Comment:
This is an environment variable to prevent Geti from collecting the VISUAL_PROMPTING TaskType and its model. It may be cleared in a future Geti integration, but for now we need this code because it's tied into the TaskType import on the Entity side.